ui/src/api/containers.js

```diff
@@ -81,6 +81,50 @@ export function startContainer(id) {
     .then(errorHandler);
 }
 
+export function restartContainer(id) {
+    const url = `/containers/${id}/restart`;
```

Comment:
nit: we'd been using 2 space tabs elsewhere, there's an `.editorconfig` in the root of the `ui/` directory that might help to configure project specific codestyle settings :smile: